doc/source/installation.rst

```diff
@@ -10,9 +10,12 @@ Use::
 
 Pip will install :
 
-1. The library and all dependencies into your site-packages directory
-2. Redfish client master conf file into /etc/redfish_client.conf
-3. Data file (templates) into /usr/share/redfish-client/templates
+1. The library and all dependencies into prefix/lib/pythonX.Y/site-packages directory
+2. Redfish client master conf file into prefix/etc/redfish_client.conf
+   Unless if prefix = '/usr', in that case force configuration file to be in /etc except
+   for Debian based system
```

Comment:
So we need to add another variable (and by that I don't necessarily mean a python variable) but a way to render variable that location. 

Typically we need:
one for TEMPLATE
one for STDCONFFILE
one for PYTHONLIB
one for BINARIES
what else ?